src/Collapsible/README.md

```diff
@@ -223,4 +229,18 @@ an **uncontrolled** Collapsible by getting a ref to the component and calling
 the component will be updated accordingly.
 
 
-`
+### Theme Variables (SCSS)
```

Comment:
The theme variables are no longer necessary in the README with the [changes recently merged](https://github.com/edx/paragon/pull/931). Just a symptom of a delay in reviewing this PR, my apologies.